hpedockerplugin/volume_manager.py

```diff
@@ -1058,6 +1078,16 @@ def mount_volume(self, volname, vol_mount, mount_id):
         else:
             mount_dir = ''
 
+        if vol['fsOwner'] is not None:
```

Comment:
you can add if condition like this
`if 'fsOwner' in vol and vol['fsOwner'] `

Reply:
Made the change as per suggestion